spec/controllers/api/v1/requested_packages_controller_spec.rb

```diff
@@ -193,81 +193,107 @@
       end
     end
 
+    def create_requested_packges_for(user, order)
+      packages = order.orders_packages.map(&:package)
+      packages.map do |p|
+        create(:requested_package, :with_available_package, package: p, user: user)
+      end
+    end
+
```

Comment:
that's kinda weird, if an order has `orders_packages` then there shouldn't be any `requested_packages`
When an order is submitted (cart_checkout), all the requested packages are added to the order, and then the requested packages are emptied